ext/reflection/php_reflection.stub.php

```diff
@@ -370,6 +370,9 @@ public function newInstanceWithoutConstructor(): object {}
     /** @tentative-return-type */
     public function newInstanceArgs(array $args = []): ?object {}
 
+    /** @tentative-return-type */
+    public function newInstanceFromData(array $data, array $args = []): ?object {}
```

Comment:
You are creating a new API, no new for the return type to be tentative.
```suggestion
    public function newInstanceFromData(array $data, array $args = []): ?object {}
```